app/celer-sim/CMakeLists.txt

```diff
@@ -133,4 +133,9 @@ celer_sim_test(lar
   lar-sphere.gdml gamma-4evt-15prim.hepmc3 lar-mctruth.root
 )
 
+if(NOT CELERITAS_CORE_GEO STREQUAL "ORANGE")
+    set_tests_properties("app/celer-sim:lar:cpu" PROPERTIES DISABLED True)
```

Comment:
Is the time frame for implementing VecGeom surface normals short enough that it would be reasonable to hold off on this until that's in (so we can continue to run optical problems with VecGeom)?

Reply:
As a temporary hack (as long as we don't care about correct answers) we could return `geo.dir()` inside the call to `geo.normal()`... so that way we always get something physical.